src/MR-Deprecation.md

```diff
@@ -43,7 +43,7 @@ Benefits:
 
 We'll use the example of the "word count" application which summarizes a set of documents into a mapping from each word to the total number of its occurrences in the documents. This involves both a mapping stage where one document is transformed into a stream of words and a reducing stage that performs a COUNT DISTINCT operation on the stream and populates a Hazelcast IMap with the results.
 
-This is the word count code in MapReduce (also available on [hazlcast-jet-code-samples](https://github.com/hazelcast/hazelcast-jet-code-samples/blob/master/core/mapreduce-migration/src/main/java/WordCountCoreApi.java)):
+This is the word count code in MapReduce (also available on [hazlcast-jet-code-samples](https://github.com/hazelcast/hazelcast-jet-code-samples/blob/v0.4/batch/mapreduce-migration/src/main/java/WordCountCoreApi.java)):
 
```

Comment:
@kwart , Is that a good idea to give a specific release number here? What if the sample in 0.4 branch becomes obsolete?

Reply:
Also I realized "hazelcast" is written as "hazlcast" in the link text, my mistake, you can copyedit it or I can do the fix after merge.

Reply:
I've fixed the typo.

WRT link to the `0.4`: It's significantly safer to provide links to tags than links to live branches.

Becoming obsolete is exactly the reason why it should point to a tag. Next Jet version could come without this exact sample, but we still will have the link to this sample in the IMDG documentation.

The original link doesn't work. The `master` branch doesn't contain the class. Current default branch is the `0.4-maintenance` which contains the class. But still, the tags are safer than branches :)